FTLGameWin32.h

```diff
@@ -4,18 +4,20 @@
 
 #define LIBZHL_API 
 
-#ifdef _WIN32
+#ifdef _MSC_VER
```

Comment:
Please note, this file is auto-generated by the parsefuncs scripts and this change will be replaced.

Also we don't use `_MSC_VER` here because we don't use MSVC.
Win32 is specifically the clang/gcc mingw32 build.

Reply:
However I think the `__declspec` stuff is still fine for GCC/Clang, at least some of it is.

Reply:
Yeah it's just a local change that I tested to get rid of the warnings. I'm aware that it's generated.

`_MSC_VER` here is to gate the `novtable` thing I've said in discord. In my test neither g++ 9.4 (mingw) nor clang 10 understood it unless clang was invoked with `-fms-compatibility` which breaks pretty much every headers in mingw instead.

Reply:
Ah alright, well if you want to change the code this is generated by, it's in [parsefuncs.lua here](https://github.com/FTL-Hyperspace/FTL-Hyperspace/blob/develop/libzhlgen/lua/parsefuncs.lua#L1434) but also this section was tweaked in the `develop` branch to add SWIG support to it, so just a heads up that might be good once you consider the Lua stuff unless you like resolving merge conflicts :smile: 

We could totally do `_WIN32` and also `__GNUC__` and/or `__clang__` to be more specific about which declspecs can be used and when or provide alternatives.